src/verify-valid-url.js

```diff
@@ -23,4 +22,21 @@ exports.verifyValidUrl = async function (req, url) {
       );
     }
   }
+}
+
+exports.verifyValidUrl = verifyValidUrl;
+
+exports.findUrlsToValidateInScopes = function (scopes) {
```

Comment:
I think this function would be simpler with for loops, or even with a reduce. I won't block approval on this, but thought I'd share an implementation that I thought made sense

```js
exports.findUrlsToValidateInScopes = function (scopes) {
  const result = []

  for (let scopeKey in scopes) {
    const scope = scopes[scopeKey]
    for (let specifier in scope) {
      const address = scope[specifier]
      // create url to validate
    }
  }
}
```

Reply:
I think that makes total sense, let me refactor :)